### PR TITLE
Fix license generator

### PR DESCRIPTION
**Pull request by oleorhagen**

### CODE REVIEW

**Comment by oleorhagen** [comment]
NOTE: Needs to be cherried to 3.2.x

**Review by lluiscampos — APPROVED** [review]

**Comment by oleorhagen** [comment]
@mender-test-bot cherry-pick to:
* 3.2.x

File: extra/license-overview-generator [hunk 1]
```diff
@@ -1508,7 +1508,7 @@ def do_repos():
             if any([file.endswith(".go") for file in files]):
                 do_go_repo(repopath)
                 break
-            elif "gulpfile.js" in files:
+            elif os.path.basename(repopath) in ["gui", "gui-enterprise"]:
```
Comment (kacf) [inline extra/license-overview-generator:1511]: Why? Is `gulpfile.js` wrong?

Comment (mzedel) [reply extra/license-overview-generator:1511]: yes: https://github.com/mendersoftware/gui/pull/1924

**Pull request merged**